src/uaclient/uasession.cpp

```diff
@@ -55,9 +57,17 @@ UaSession::~UaSession ()
 
 void UaSession::clientRunIterateThread(void)
 {
-    while(m_clientRunIterateToggle) {
-        UA_Client_run_iterate(m_client, 1000);
+    
+    while(m_clientRunIterateToggle) 
+    {
+        std::this_thread::sleep_for(std::chrono::milliseconds(500));
+        UaStatus status = UA_Client_run_iterate(m_client, 1000);
+        if( !status.isGood() )
+        {
+            LOG(Log::ERR) << "in UaSession::clientRunIterateThread() " << status.toString().toUtf8();
```

Comment:
I guess the log component is missing ?

Reply:
Not, the UaSession class logs without a specific log component.

Reply:
I see, we just did not unify it:
https://github.com/quasar-team/open62541-compat/blob/master/src/logit_logger.cpp#L19
okay, so merge it please and we can unity some other days ;-)